pyproject.toml

```diff
@@ -87,7 +87,7 @@ select = ["I", "E", "W", "D", "PL", "F"]
 # Rules considered over other incompatible ones
 exclude = ["docs/conf.py", "setup.py"]
 
-ignore = ["D407", "D203", "D213", "D416", "PLR0912", "PLR0911", "PLR0915", "PLR2004", "PLR0913", "F811"]
+ignore = ["D407", "D203", "D213", "D416", "PLR0912", "PLR0911", "PLR0915", "PLR2004", "PLR0913", "F811", "E203"]
```

Comment:
Add a comment below in the `toml` file for why this rule is ignored. You can see the comments for the other rules.